apps/bfd-data-npi/Jenkinsfile

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent {
+        kubernetes {
+            yaml """
+apiVersion: v1
+kind: Pod
+spec:
+  serviceAccount: bfd
+  containers:
+  - name: bfd-cbc-build
+    image: public.ecr.aws/c2o1d8s9/bfd-cbc-build:jdk11-mvn3-an29-tfenv # TODO: make sure this is coming from somewhere sane
+    command:
+    - sleep
+    args:
+    - 99d
+    resources:
+      limits:
+        memory: 8192Mi
+        cpu: 4000m
+      requests:
+        memory: 8192Mi
+        cpu: 4000m
+"""
+        }
+    }
+     triggers {
+        cron('0 0 28 * *')
```

Comment:
more curious, and for my edification....why do we have a cron job if this is part of a build?